services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -648,6 +660,31 @@ export const RateDetails = ({
                                                                 }
                                                             />
                                                         </FormGroup>
+                                                        {ratesAcrossSubmissions && (
+                                                            <FormGroup>
+                                                                <Checkbox
+                                                                    id={`sharedRateCheckBox-${index}`}
+                                                                    name={`rateInfos.${index}.sharedRateCert`}
+                                                                    label="This rate certification was uploaded to another submission."
+                                                                    aria-required={
+                                                                        false
+                                                                    }
+                                                                    checked={
+                                                                        rateInfo.sharedRateCert
+                                                                    }
+                                                                    onChange={(
+                                                                        e
+                                                                    ) =>
+                                                                        setFieldValue(
+                                                                            e
+                                                                                .target
+                                                                                .name,
```

Comment:
This checkbox is only used for toggling the UI work in [MR-2685](https://qmacbis.atlassian.net/browse/MR-2685), linking to other submissions. 

The thought here was to have a value in `formik` to track the checkbox on each rate cert and toggle on without modifying the new `packagesWithSharedRateCerts` field. Thew new `packagesWithSharedRateCerts` will be storing the linked submission ID's and saved to the health plan form data so I was trying to avoid using this field as a toggle, although it is still used to set the initial value of `sharedRateCert`.

I believe this means that [MR-2685](https://qmacbis.atlassian.net/browse/MR-2685) will be blocking [MR-2684](https://qmacbis.atlassian.net/browse/MR-2684).  I could saved `sharedRateCert` field into the health plan form data, but I thought it was an unnecessary and redundant field once MR-2685 was finished. Then we could use `packagesWithSharedRateCerts` to display the `Shared` tag on the summary and review/submit pages

Thoughts? If this is good, then I can update the sprint by switching out MR-268 with MR-2685?

Reply:
I think the checkbox makes sense, and it's fine to wait on 2684 until 2685 is finished.  Just to check my understanding:
The state of the checkbox determines whether we should see a "please select the shared submissions" input (2685), and once there are shared submissions, because they're available throughout the app with the form data, we can use the shared submissions array to determine whether we show a "shared" tag (2684).

Reply:
💯!